src/modules/Get.js

```diff
@@ -0,0 +1,11 @@
+export const id = 'ZTp6WjEdBYQQYfIGvFXP';
+export const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`;
+
+export default async function getUsers() {
```

Comment:
[REQUIRED] In this milestone it is a requirement to make use of arrow functions. Kindly refactor your code to satisfy this condition.

Reply:
thank you.